apsi-enrollment-fe/src/app/modules/main/pages/all-payments/all-payments.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AllPaymentsComponent } from './all-payments.component';
+
+describe('AllPaymentsComponent', () => {
+  let component: AllPaymentsComponent;
+  let fixture: ComponentFixture<AllPaymentsComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
```

Comment:
You should define mocked SpyObj for PaymentService here. As there are actually no tests here you can also remove that file for now (as an alternative for adding SpyObj).

Reply:
removed